mq/v2_v3_migration_guide/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: IronMQ v3 Migration Guide
+summary: "The complete guide to moving your app from IronMQ v2 to IronMQ v3"
```

Comment:
complete sounds a little pompous considering it's 50 lines :) Just my opinion but I'd remove it.